src/MSBuild/XMake.cs

```diff
@@ -2624,7 +2624,7 @@ private static void StartLocalNode(CommandLineSwitches commandLineSwitches)
 
                         // If FEATURE_NODE_REUSE is OFF, just validates that the switch is OK, and always returns False
                         bool nodeReuse = ProcessNodeReuseSwitch(commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.NodeReuse]);
-                        bool lowpriority = commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.LowPriority][0].Equals("true");
+                        bool lowpriority = commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.LowPriority].Length > 0 ? commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.LowPriority][0].Equals("true") : false;
```

Comment:
Might be good to factor out commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.LowPriority]. Also, if they use it, but they use it wrong, do you think it's more likely they were trying to turn it on or off?